### PR TITLE
design fix: turn conversation background grey

### PR DESCRIPTION
**Pull request by johnxnguyen**
- Change conversation background colour to grey (see screenshot)

conversation background color

### CODE REVIEW

**Comment by daehn** [comment]
Let's wait for Circle-CI to finish before merging.

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/TextMessageCell.m [hunk 1]
```diff
@@ -98,7 +98,7 @@ - (void)createTextMessageViews
     ColorScheme *scheme = ColorScheme.defaultColorScheme;
     self.messageTextView.editable = NO;
     self.messageTextView.selectable = YES;
-    self.messageTextView.backgroundColor = [scheme colorWithName:ColorSchemeColorBackground];
+    self.messageTextView.backgroundColor = [scheme colorWithName:ColorSchemeColorBackgroundNew];
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/TextMessageCell.m]: This seems like temporary name. Would nice to if it was more descriptive.

**Comment by daehn** [comment]
@johnxnguyen looks like a couple of tests failed on Circle-CI, can you check them?

**Comment by johnxnguyen** [comment]
having a look now

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/TextMessageCell.m (hunk 1 above)
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/TextMessageCell.m]: this was already in `ColorScheme`, though I'll update it

Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/TextMessageCell.m]: I also saw it was already like that. Would still be nice to clean up.

**Pull request merged**